config/security.yml.example

```diff
@@ -1,6 +1,5 @@
 production: &default
-  # replace this with a random string of at least 20 characters
-  encryption_key: 12345
+  encryption_key: facdd3a131ddd8988b14f6e4e01039c93cfa0160
```

Comment:
@rohansharmasitoula don't hardcode the encryption_key, rotate it and move it to env